### ISSUE TITLE
ApI Key

### ISSUE THREAD

**Issue by MindlessXD** [Rei-x/discord-speech-recognition#30]
there is a google api key included. is this being used for default package implementation

**Comment by Rei-x** [comment]
Yes, it's from python speech_recognition package and it's for testing purposes only, you can read more here: https://github.com/Uberi/speech_recognition/blob/c89856088ad81d81d38be314e3db50905481c5fe/speech_recognition/__init__.py#L850
